src/Steps/Release/PlanRelease.php

```diff
@@ -404,45 +404,78 @@ protected function reviewLibraryVersion(
         InputInterface $input,
         LibraryRelease $selectedVersion
     ) {
-        $question = new Question(
-            "Please enter a new version to release for <info>"
-            . $selectedVersion->getLibrary()->getName() . "</info>: ",
-            $selectedVersion->getVersion()
-        );
-        $newVersionName = $this->getQuestionHelper()->ask($input, $output, $question);
+        $versions = [
+            'new_version' => [
+                'text' => 'Please enter a new version to release for <info>%s</info>: ',
+                'default' => $selectedVersion->getVersion(),
+            ],
+            'prior_version' => [
+                'text' => 'Optional: modify the prior version for <info>%s</info>: ',
+                'default' => $selectedVersion->getPriorVersion(),
+            ],
+        ];
+
+        foreach ($versions as $key => $options) {
+            $question = new Question(
+                sprintf($options['text'], $selectedVersion->getLibrary()->getName()),
+                $options['default']
+            );
+            $result = $this->getQuestionHelper()->ask($input, $output, $question);
+
+            // If version is invalid, show an error message
+            if (!Version::parse($result)) {
+                $this->log(
+                    $output,
+                    "Invalid version {$result}; Please enter a tag in w.x.y(-[rc|alpha|beta][z]) format",
+                    "error"
+                );
+                $this->reviewLibraryVersion($output, $input, $selectedVersion);
```

Comment:
I think it makes more sense to return to the plan if there's an issue here? Maybe you'll just skip this and come back, right now you have to find some valid input to get back.

Reply:
Hmmm, I'm not sure I agree. I think the flow would be that you'll want to put a valid tag in since you've already chosen that module from the list, but I DO think that pressing enter with no entry after this should take you back to the plan - currently it throws a warning (as per your message below) and takes you back to the question again.

You OK if I make it so that an empty option takes you back to the plan and leave it there?

Reply:
Yeah that's good. I think the situation I was in was I had incorrectly chosen a module. I can quite remember...

Reply:
It was definitely broken